### ISSUE TITLE
An ugly blank leads each comment

### ISSUE THREAD

**Issue by kwankyu** [sagemath/trac-to-github#107]
As seen in 

https://34.105.185.241/sagemath/sage-import-3/issues/30158#comment:2

there is a visible blank at the beginning of each comment. The blank is the space for the anchor inserted for reference. As this looks ugly, it would be better to actually show the anchor. The code implementing this will be:
```diff
     note = preamble + note
 
     if comment_id:
-        anchor = f"<a id='comment:{comment_id}'></a>"
+        anchor = f"<a id='comment:{comment_id}'>**comment:{comment_id}**</a>"
         note = anchor + '\n' + note
 
     if dest is None : return
```

I could not test this because running `./migrate.py` fails for me with the message
```python
...
  File "/usr/local/lib/python3.10/site-packages/github/Repository.py", line 968, in create_repository_file
    from github.RepositoryFile import RepositoryFile
ModuleNotFoundError: No module named 'github.RepositoryFile'
```
even after reinstalling `pygithub`...

**Comment by mkoeppe** [comment]
The missing `RepositoryFile` is now there

**Comment by mkoeppe** [comment]
I agree that it's a good idea to use this blank line for the comment number. Though the comment links are rewritten to use the fragment URL, at least on some tickets, ticket comments refer to earlier ticket comments by number without using the correct format.

**Comment by mkoeppe** [comment]
But perhaps if it is followed by the "Replying to [...]" message, the "\n" can be suppressed?

**Comment by mkoeppe** [comment]
Fixed by #109
